Jenkinsfile

```diff
@@ -10,10 +10,18 @@ pipeline {
             }
           }
           steps {
+			sh '''
+				pip freeze |grep flake8
+				virtualenv venv
+				source venv/bin/activate
+			'''
+			  
             echo "Installing Insights..."
             sh 'pip install --user -e .[develop]'
```

Comment:
@eduardocerqueira you are activating a virtualenv but that won't be used for the other `sh` commands since the env would be reset.

Reply:
Hi @bfahr thx for the comment. @YakovLittle can you please take a look at this?

Reply:
@YakovLittle @eduardocerqueira I believe this will work if you want to use the virtualenv:
```
          steps {
            echo "Installing Insights..."
            sh """
                virtualenv -p /usr/bin/python2 .env
                source .env/bin/activate
                pip install -e .[develop]
            """
            echo "Testing with Pytest..."
            sh """
                source .env/bin/activate
                pytest
            """
          }
```

Reply:
Actually `--user` doesn’t work when a virtual environment is active.